### PR TITLE
Add a command line option to store logs into a separate file

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Doc/Design
- [ ] Unit test

#### What is the motivation for this PR?
Support store logs into a separate log file.

#### How did you do it?
Add an option `-e` or `--extra_log_file`, if present, stores the logs from `linkmgrd` into `/var/log/mux/linkmgrd.log`.

#### How did you verify/test it?
Run `linkmgrd` with the option on/off

### CODE REVIEW

File: src/common/MuxLogger.cpp [hunk 1]
```diff
@@ -99,12 +100,13 @@ void MuxLogger::initialize(
...
     boost::log::init_from_settings(settings);
 
-//    boost::filesystem::remove(path);
-
-//    boost::log::add_file_log(
-//        keywords::file_name = path,
-//        keywords::format = "[%TimeStamp%] [%Severity%] %Message%"
-//    );
+    if (extraLogFile) {
+        boost::filesystem::remove(path);
```
Comment (zjswhhh) [inline src/common/MuxLogger.cpp:104]: Do we want to add some protection on this `remove()`? A try/catch maybe?

Comment (lolyu) [reply src/common/MuxLogger.cpp:104]: If the file doesn't exist, `remove` will return `false`, for those exceptions that `remove` might throw(rw issue, OS error), we don't have to catch them, right?

Comment (zjswhhh) [reply src/common/MuxLogger.cpp:104]: I guess it's fine, I assume we are using this argument only for dev/debug purpose.

**Review by zjswhhh — APPROVED** [review]
Looks good to me!

**Pull request merged**